docs/getting-started/supported-languages-and-tools.md

```diff
@@ -395,7 +395,7 @@ The table below lists all languages and frameworks that Codacy supports and the
 
 Codacy adds support for new languages and tools by using [a Docker image to run each tool](https://github.com/codacy/codacy-example-tool).
 
-The following table lists the Codacy GitHub repositories corresponding to each supported tool. Use these repositories to check the version and extra plugins supported by each tool, or to submit GitHub issues related to each tool.
+The following table lists the Codacy GitHub repositories corresponding to each supported tool. Use these repositories to check the extra plugins supported by each tool or to submit GitHub issues related to each tool. To learn more about the tool versions used by Codacy, see the latest [release notes](../release-notes/index.md).
```

Comment:
Should we extend this edit to the self-hosted versions of the page?

Note: for Cloud, we moved this content to this page from the old "Codacy plugin tools" page (see https://github.com/codacy/docs/pull/1637), which still exists for SH versions, e.g. https://docs.codacy.com/v10.0/related-tools/codacy-plugin-tools/.

Reply:
I tend to prefer longer, more descriptive/actionable link texts. What do you think?

```suggestion
The following table lists the Codacy GitHub repositories corresponding to each supported tool. Use these repositories to check the extra plugins supported by each tool or to submit GitHub issues related to each tool. To learn more about the tool versions used by Codacy, [see the latest release notes](../release-notes/index.md).
```